### DISCUSSION TITLE
这种是什么问题，会影响抢购吗？

### DISCUSSION THREAD

**Discussion by Lrrrrrrt** [mikumifa/biliTickerBuy#120] [提问专区] [answered]
![屏幕截图 2024-06-16 013632](https://github.com/mikumifa/biliTickerBuy/assets/108092864/da04dde1-daa8-4242-9b78-51e8db6d25b5)

**Comment by Lrrrrrrt** [comment]
过完验证码之后就会出现这种情况

**Comment by WittF** [comment, marked as answer]
不影响，这是gradio无法找到frp客户端的提示，而frp的作用是内网穿透，不影响抢票的

**Reply by Lrrrrrrt** [reply]
好的 谢谢
